lib/faker/app.ex

```diff
@@ -19,13 +19,41 @@ defmodule Faker.App do
   end
 
   @doc """
-  Return app name.
+  Returns a SemVer version.
+
+  ## Options:
+
+  * `:allow_pre` - when `true`, a pre-release identifier (e.g.: `-dev`)
+    will be appended (default: `true`)
+  * `:allow_build` - when `true`, a build identifier (e.g.: `+001`)
+    will be appended (default: `true`)
+  """
+  @spec semver(Keyword.t) :: String.t
+  def semver(opts \\ []) do
+    allow_pre = Keyword.get(opts, :allow_pre, true)
+    allow_build = Keyword.get(opts, :allow_build, true)
```

Comment:
yup, I was on a fence and ended up using a default that will show people a broader range of values, but I guess you're right that's it's more common the other way around.

Fixed in 5d81e85915da2fe27a84548ac35edfdc2b03b414

PS thanks for this project!